streaming/base/converters/jsonToMDS.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""A utility to convert a json dataset to MDS."""
+
+from typing import Any, Callable, Dict, Optional
+
+from pyspark.sql import SparkSession
+from pyspark.sql.types import StructType
+
+from streaming.base.converters.dataframeToMDS import dataframeToMDS
+
+
+def jsonToMDS(input_path: str,
+              schema: StructType,
+              merge_index: bool = True,
+              sample_ratio: float = -1.0,
+              mds_kwargs: Dict[str, Any] = {},
+              udf_iterable: Optional[Callable] = None,
+              udf_kwargs: Dict[str, Any] = {}):
```

Comment:
Default arg must not be a mutable type, please default it to `None` and normalize to empty dict inside the method.